numpy/__init__.py

```diff
@@ -109,8 +109,7 @@
 import sys
 import warnings
 
-from ._globals import ModuleDeprecationWarning, VisibleDeprecationWarning
-from ._globals import _NoValue
+from ._globals import ModuleDeprecationWarning, VisibleDeprecationWarning , _NoValue
```

Comment:
Could you put this in a tuple? We try to keep lines < 80 characters. Something like
```
from ._globals import (
    ModuleDeprecationWarning, VisibleDeprecationWarning , _NoValue
)
```

Reply:
Of course, at this point it uses more lines than it did before...

Reply:
Still slightly better that way. Let's just get this in to not spend more time on style issues.